smoke-tests/apps/ConnectionStringOverrides/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/ConnectionStringOverridesTest.java

```diff
@@ -98,9 +98,9 @@ private static void testApp(String iKey) throws Exception {
     // and metrics too
 
     List<Envelope> clientMetrics =
-        testing.mockedIngestion.waitForMetricItems("http.client.request.duration", 1);
+        testing.mockedIngestion.waitForStandardMetricItems("http.client.request.duration", 1);
```

Comment:
maybe pass in the `_MS.MetricId` as the arg, since that's the important part from the breeze perspective?

Reply:
https://github.com/microsoft/ApplicationInsights-Java/pull/3676/files#r1594769442